### PR TITLE
File management + cv2

### PR DESCRIPTION
**Pull request by KimNikita**

### CODE REVIEW

**Comment by KimNikita** [comment]
done issue #2 and #9

File: src/samples/sample_sounds.py [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
-        print('Now play B4')
-        k.play_sound()
-
-
-if __name__ == '__main__':
-    sys.exit(main() or 0)
+import os
+import sys
+sys.path.append("..")
+from piano_key import PianoKey
```
Comment (DumDereDum) [inline src/samples/sample_sounds.py]: add `modules.`

File: src/samples/sample_sounds.py [hunk 2]
```diff
@@ -1,34 +1,35 @@
...
+
+
+def main():
+    path = os.path.abspath('sound_7')
```
Comment (DumDereDum) [inline src/samples/sample_sounds.py]: the **path** is equal to `src/samples/sound_7/`, but we need `src/sounds/sounds_7`
possible fix:
`path = os.path.abspath('')[:-7] + 'sounds\sound_7' `

Comment (DumDereDum) [reply src/samples/sample_sounds.py]: why `sound_7` but no `sound_4`

Comment (KimNikita) [reply src/samples/sample_sounds.py]: because there are 7 notes

**Review by DumDereDum — APPROVED** [review]
👍

**Pull request merged**